conans/test/integration/run_envronment_test.py

```diff
@@ -37,12 +41,16 @@ def build(self):
         self.assertIn("Hello Hello0", client.out)
 
     def test_shared_run_environment(self):
-        client = TestClient()
+        servers = {"default": TestServer()}
+        client = TestClient(servers=servers, users={"default": [("lasote", "mypass")]})
         cmake = """set(CMAKE_CXX_COMPILER_WORKS 1)
 set(CMAKE_CXX_ABI_COMPILED 1)
 project(MyHello CXX)
 cmake_minimum_required(VERSION 2.8.12)
-
+if(APPLE)
```

Comment:
better to remove this to really test that a broken rpath doesn't affect while the linker finds the libs in other place